components/studio/bin/hab-studio.ps1

```diff
@@ -455,6 +455,17 @@ function Set-Secrets {
   }
 }
 
+function Update-SslCertFile {
+  if($env:SSL_CERT_FILE) {
+    try {
+      $cert_filename = (Get-Item $env:SSL_CERT_FILE).Name
+      $env:SSL_CERT_FILE = Join-Path "Habitat:\hab\cache\ssl" $cert_filename
```

Comment:
similar to linux it wil be scoped for the current process and all child processes. When you enter a studio it starts a new pwsh process so this will not leak into the parent (non-studio) environment.